src/ruvsarpur.py

```diff
@@ -20,18 +20,18 @@
 #      For alternative install http://stackoverflow.com/a/33163704
 
 
-import sys, pprint, os.path, re
+import sys, os.path, re
 import textwrap # For text wrapping in the console window
 from colorama import init, deinit # For colorized output to console windows (platform and shell independent)
-from termcolor import colored, cprint # For shorthand color printing to the console, https://pypi.python.org/pypi/termcolor
+from termcolor import colored # For shorthand color printing to the console, https://pypi.python.org/pypi/termcolor
 from pathlib import Path # to check for file existence in the file system
 import json # To store and load the tv schedule that has already been downloaded
 import argparse # Command-line argument parser
 import requests # Downloading of data from HTTP
 import datetime, dateutil.relativedelta # Formatting of date objects and adjusting date ranges
 from xml.etree import ElementTree  # Parsing of TV schedule XML data
 from fuzzywuzzy import fuzz # For fuzzy string matching when trying to find programs by title or description
-from operator import itemgetter, attrgetter # For sorting the download list items https://docs.python.org/3/howto/sorting.html#operator-module-functions
+from operator import itemgetter # For sorting the download list items https://docs.python.org/3/howto/sorting.html#operator-module-functions
```

Comment:
It is generally not good practice to add new refactorings in revisions. We'll leave this for now though.